zephyr/Makefile

```diff
@@ -71,15 +70,10 @@ $(GENERIC_TARGETS): $(LIBMICROPYTHON)
 $(CLEAN_TARGETS):  clean
 
 $(GENERIC_TARGETS) $(KCONFIG_TARGETS) $(CLEAN_TARGETS):
-	$(RM) -f outdir/$(OUTDIR_PREFIX)/zephyr.lnk
```

Comment:
Ok, I posted updated patch which addresses this. Testes all of: 1) build from scratch; 2) empty build afterwards; 3) touching a file and building. Behaves as expected (links in 1) and 3) cases, minimal extra command noise in 2) case).